### DISCUSSION TITLE
Změnit datum v oblíbených

### DISCUSSION THREAD

**Discussion by rblaha15** [jaro-jaro/DPMCB#146] [Ideas]
Pokud v Oblíbených klikneš na spoj, co nejede dnes, navrhuju, aby se změnilo datum

Aktuálně je to totiž jediný způsob jak se dostat do detailu spoje, co nejede dnes

Navíc když už se tam dostaneš a pak si z něj dáš odjezdy, ukazuje ti to ndešní odjezdy, co by nemělo

**Comment by 0ndra0** [comment]
jo, to mi přijde dobrý
